www-client/luakit/metadata.xml

```diff
@@ -2,4 +2,20 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
   <!-- maintainer-needed -->
-  </pkgmetadata>
+  <maintainer type="person">
+    <email>annulen@yandex.ru</email>
+    <name>Konstantin Tokarev</name>
+  </maintainer>
```

Comment:
Who is this person, and why are **YOU** adding them here? They should volunteer by themself! 
And in that regard, we don't let people become maintainers for packages they haven't contributed to. In other words, hoard it without taking care of it.